src/main/resources/artifact-ignores.properties

```diff
@@ -135,6 +135,33 @@ git-3.0.2
 git-3.0.3
 workflow-multibranch-2.10
 pipeline-github-lib-1.0
+# The components below are being prememptively blocked so that we can 
+# synchronize their release. To be followed by PR to unblock when we're ready.
+scm-api-2.0.2
+git-2.6.3
+git-3.0.4
+branch-api-2.0.2
+bitbucket-branch-source-2.0.2
+github-branch-source-plugin-2.0.1
+cloudbees-folder-5.17
+workflow-multibranch-2.11
+blueocean-1.0.0-b22
+blueocean-parent-1.0.0-b22
+blueocean-web-1.0.0-b22
+blueocean-dashboard-1.0.0-b22
+blueocean-personalization-1.0.0-b22
+blueocean-rest-1.0.0-b22
+blueocean-commons-1.0.0-b22
+blueocean-i18n-1.0.0-b22
+blueocean-jwt-1.0.0-b22
+blueocean-rest-impl-1.0.0-b22
+blueocean-pipeline-api-impl-1.0.0-b22
+blueocean-config-1.0.0-b22
+blueocean-github-pipeline-1.0.0-b22
+blueocean-git-pipeline-1.0.0-b22
+blueocean-display-url-1.0.0-b22
```

Comment:
@recampbell 
My bad, please remove, they are not part of blueocean release, are separate standalone plugins.
blueocean-display-url-1.0.0-b22
blueocean-autofavorite-1.0.0-b22